tests/tests.rs

```diff
@@ -233,14 +233,24 @@ fn compare_generated_header(
             }
         }
 
-        // Overwrite the expectation with actual output.
-        if env::var_os("BINDGEN_OVERWRITE_EXPECTED").is_some() {
-            let mut expectation_file =
-                fs::File::create(looked_at.last().unwrap())?;
-            expectation_file.write_all(actual.as_bytes())?;
+        if let Some(var) = env::var_os("BINDGEN_OVERWRITE_EXPECTED") {
+            if var == "1" {
+                // Overwrite the expectation with actual output.
+                let mut expectation_file =
+                    fs::File::create(looked_at.last().unwrap())?;
+                expectation_file.write_all(actual.as_bytes())?;
+            } else { //usecase: var = "meld" -> You can hand check differences
```

Comment:
While we're here, might be worth to make this `} else if var != "0" {`, so that `OVERWRITE_EXPECTED=0` behaves as expected.

Reply:
Fixed in newest commit.